.changeset/moody-moose-smash.md

```diff
@@ -0,0 +1,7 @@
+---
+"@jpmorganchase/uitk-core": minor
+"@jpmorganchase/uitk-lab": minor
+"@jpmorganchase/uitk-theme": minor
+---
+
+changes to text fontsize tokens
```

Comment:
Need more detail on what is the change, especially for core package, would be better to say before and after